client/src/components/Chart.jsx

```diff
@@ -1,103 +1,54 @@
-import React, { useEffect, useRef, useCallback, useState } from 'react';
-import { useSelector, connect } from 'react-redux';
-import * as chartJs from 'chart.js';
+import React, { useState } from 'react';
+import { ResponsiveContainer, LineChart, Line, XAxis, YAxis, CartesianGrid, Tooltip, Legend } from 'recharts';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
 const Chart = props => {
-  const { chartType, observations } = props;
-  const temperatureData = useSelector(state => state.temperatureReducer.temperature);
-  const [chartConfig, setChartConfig] = useState({
-    type: chartType,
-    data: {
-      labels: temperatureData.labels,
-      datasets: [
-        {
-          label: 'Patient temperature',
-          backgroundColor: '#4760ff',
-          data: temperatureData.values,
-        },
-      ],
-    },
-    options: {
-      scales: {
-        yAxes: [
-          {
-            ticks: {
-              max: 110,
-              min: 80,
-            },
-          },
-        ],
-      },
-    },
+  const { observations } = props;
+
+  const temps = observations.map(observation => {
+    const datetime = new Date(observation.date).toISOString().slice(0, 10);
+    let temp;
+    if (
+      !observation.physical.feverSeverity ||
+      observation.physical.feverSeverity === '' ||
+      !parseFloat(observation.physical.feverSeverity, 10) > 0
```

Comment:
Is this just for redundancy? For the time being we can leave it because I'm fine with making sure we have all our bases covered, but just for future reference I want to be sure I'm clear

Reply:
Yeah, I don't like some of this stuff

The `!parseFloat > 0` is confusing - it's `!`ing a `number` and comparing it to `0` so `boolean > 0`

Also, I don't see `observation.physical.feverSeverity` ever being anything other than a `number`/`''`/`undefined`/`null` - which should be covered with the first `!observation.physical.feverSeverity`

**However,** it's probably fine and won't break anything

Reply:
I agree, it is really confusing. Its from the previous author. I recognized it was doing validation and decided to leave it. I can refactor if this is a big issue

Reply:
Also, the first 2 screenshots for after are mobile ;) .

Reply:
It shouldn't be a problem